roundtables/rails-week4/day1-session-media-ranker.md

```diff
@@ -0,0 +1,38 @@
+# Exercism: Session and Media Ranker
+
+Today we are going to be reviewing Session and discussing the current progress on Media Ranker.
+
+For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer. You are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+### Resources
+
+* [Session](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/session.md)
+* [Testing Session](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/testing-session.md)
+* [Media Ranker](https://github.com/Ada-C13/media-ranker)
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+### Session
+
+1. As a group, come up with a description for how `session` helps with log-in functionality
+1. How would our controller tests be affected by log-in functionality?
+1. How does someone "clear" the contents of `session`...
+    - in our Rails code?
+    - in our browser?
+1. Share code: If anyone has an example of `session` being used in a controller action (even if it's not working or in-progress), share with the group! Practice talking through your code. Feel free to use this time to ask questions, especially if you don't have `session` working.
```

Comment:
Didn't we do this last week?

Reply:
Yeah... Honestly I just kept this in here because the original plan was to have testing session over the weekend, and the roundtables I led on Friday were not caught up with session anyway. I'm happy to just remove this section if that's better :D

Reply:
We can keep it in as it's a tougher topic, especially testing.